src/vscp/drivers/level1/usb2can/usb2can/win32/usb2candrv.cpp

```diff
@@ -103,8 +103,7 @@ extern "C" long CanalOpen( const char *pDevice, unsigned long flags )
 				   theApp->removeDriverObject( h );
 				   h = 0;
 				}
-			}		
-	}
```

Comment:
At least to work! ;-)

Andreas Merkle skrev den 2015-05-11 22:19:
>
> In src/vscp/drivers/level1/usb2can/usb2can/win32/usb2candrv.cpp 
> <https://github.com/grodansparadis/vscp_software/pull/137#discussion_r30077318>:
>
> > @@ -103,8 +103,7 @@ extern "C" long CanalOpen( const char *pDevice, unsigned long flags )
> >  				   theApp->removeDriverObject( h );
> >  				   h = 0;
> >  				}
> > -			}		
> > -	}
>
> This } was deleted, but is necessary to compile. Isn't it?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/grodansparadis/vscp_software/pull/137/files#r30077318>.
>

-- 
Be Hungry - Stay Foolish!
Åke Hedman, Grodans Paradis AB / Paradise of the Frog
Brattbergavägen 17, 82050 LOS, SWEDEN
Phone: +46 (0)840011835 CellPhone: +46 (0)730-533146
http://www.grodansparadis.com http://www.vscp.org



---
Detta e-postmeddelande har sökts igenom efter virus med antivirusprogram från Avast.
http://www.avast.com